H/HelFEM/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+using BinaryBuilder, Pkg
+
+name = "HelFEM"
+version = v"0.0.1"
+sources = [
+    GitSource("https://github.com/mortenpi/HelFEM.git", "d3b9ce4d34fcce3296a80aef0ed9b60ce45319c9")
+]
+
+script = raw"""
+cp -v ${WORKSPACE}/srcdir/HelFEM/julia/CMake.system ${WORKSPACE}/srcdir/HelFEM/CMake.system
+
+# Set up some platform specific CMake configuration. This is more or less borrowed from
+# the Armadillo build_tarballs.jl script:
+#   https://github.com/JuliaPackaging/Yggdrasil/blob/48d7a89b4aa46b1a8c91269bb138a660f4ee4ece/A/armadillo/build_tarballs.jl#L23-L52
+#
+# We need to manually set up OpenBLAS because FindOpenBLAS.cmake does not work with BB:
+if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
+    OPENBLAS="${libdir}/libopenblas64_.${dlext}"
+else
+    OPENBLAS="${libdir}/libopenblas.${dlext}"
+fi
+
+# On i686, it seems that we need to explicitly tell GCC that we want SSE2. Refs:
+#   https://github.com/JuliaLang/julia/blob/v1.4.1/src/atomics.h#L8-L10
+#   https://stackoverflow.com/questions/16410149/error-sse2-instruction-set-not-enabled-when-including-emmintrin-h
+if [[ "${target}" == i686* ]]; then
+    export CXXFLAGS="-msse -msse2 -msse3"
+fi
```

Comment:
Ok, I got all the builds to complete by explicitly enabling SSE2 in i686. Whether that is a sensible thing to do is another question..

Reply:
I've also run into this in PR #1948; I wonder if there is something we can do in `libjulia_jll` to make this unnecessary (otherwise all packages using it will need it). The same could of course also be asked about `Julia_jll`, but I hope we'll soon be able to migrate everything away from that.

Reply:
Why SSE3?  In the linked guard I can see only SSE2

Reply:
Because I copied it without thinking from StackOverflow :sweat_smile: That said, you are probably right that we don't need that. Maybe it will even get rid of [these warnings about requiring `prescott`](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=6809&view=logs&j=8b71498c-f301-537f-b5ca-249ef5f2259f&t=5f39302e-a855-53dc-855d-a9fbde1a2c10&l=396).

```suggestion
# On i686, it seems that we need to explicitly tell GCC that we want SSE2. Refs:
#   https://github.com/JuliaLang/julia/blob/v1.4.1/src/atomics.h#L8-L10
#   https://stackoverflow.com/questions/16410149/error-sse2-instruction-set-not-enabled-when-including-emmintrin-h
if [[ "${target}" == i686* ]]; then
    export CXXFLAGS="-msse -msse2"
fi
```

Reply:
> Maybe it will even get rid of these warnings about requiring `prescott`.

It looks like it did!